app/assets/javascripts/pretty-text/addon/engines/discourse-markdown-it.js

```diff
@@ -253,7 +253,12 @@ function renderImageOrPlayableMedia(tokens, idx, options, env, slf) {
     }
   }
 
-  token.attrs[token.attrIndex("alt")][1] = altSplit.join("|");
+  const altValue = altSplit.join("|").trim();
+  token.attrSet("alt", altValue);
```

Comment:
I don't it's useful to have an empty `alt` attribute, so if `altValue` is empty, we can skip this line, maybe?

Reply:
Unfortunately there is another place that sets the `alt` value even if it is empty. I don't know where it is coming from, but I will make the change here anyway.